src/main/java/net/dv8tion/jda/api/utils/MarkdownSanitizer.java

```diff
@@ -188,6 +188,76 @@ public static String escape(@Nonnull String sequence, int ignored)
                 .compute(sequence);
     }
 
+    /**
+     * Escapes every single markdown formatting token found in the provided string.
+     * <br>Example: {@code escape("**Hello _World_", true)}
+     *
+     * @param sequence
+     *        The string to sanitize
+     * @param single
+     *        Whether it should scape single tokens or not.
+     *
+     * @throws java.lang.IllegalArgumentException
+     *         If provided with null sequence
+     *
+     * @return The string with escaped markdown
+     */
+    @Nonnull
+    public static String escape(@Nonnull String sequence, boolean single)
+    {
+        Checks.notNull(sequence, "Input");
+        if(single)
+        {
+            StringBuilder builder = new StringBuilder();
+            boolean escaped = false;
+            for (int i = 0; i < sequence.length(); i++)
+            {
+                char current = sequence.charAt(i);
+                if(!escaped)
+                {
+                    switch (current)
+                    {
+                    case '*':
+                        builder.append("\\*");
+                        break;
+                    case '_':
+                        builder.append("\\_");
+                        break;
+                    case '`':
+                        builder.append("\\`");
+                        break;
+                    case '|':
+                        builder.append("\\|");
+                        break;
+                    case '~':
+                        builder.append("\\~");
+                        break;
+                    case '>':
```

Comment:
How does this prevent escaping a message like `hello 7 > 5` as `hello 7 \> 5`? What about `>hello` without a space, which would not be a quote?

Reply:
It doesn't, but as of now, the behavior of Discord allows to escape markdown characters even if they are not part of a valid Markdown token that modifies the text, it just displays it as a character without the `\`.

Reply:
i.e.
This: 
![image](https://user-images.githubusercontent.com/7455707/127836100-9d851216-7a29-4cc0-9e92-39638da3be40.png)
Generates this:
![image](https://user-images.githubusercontent.com/7455707/127836145-8c144b59-f46b-4771-8cb1-f190e2357690.png)

Reply:
This might be acceptable in some cases, but not if the message contains a link:

![image](https://user-images.githubusercontent.com/18090140/127838407-db621648-f8f0-4b4b-a610-9b31b312a489.png)

![image](https://user-images.githubusercontent.com/18090140/127838436-16a34a07-1d3f-4ded-9eb5-df513e8cff7e.png)